communitiesuk/funding-service-design-frontend
# FS-2542 Updating contact us page to match gov uk design system

### Change description
Updated the partial page used to render contact details to match the gov-uk design system https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/

Updated translations as the headings for email, telephone etc no longer include a `:`

Translations pending for 'except public holidays' but this will be picked up in [FS-2532](https://digital.dclg.gov.uk/jira/browse/FS-2532) when the rest of the contact us page is fully translated.

- [n/a] Unit tests and other appropriate tests added or updated
- [n/a] README and other documentation has been updated / added (if needed)


### How to test
Run locally using the docker runner and navigate to http://localhost:3008/contact_us to see the changes


### Screenshots of UI changes (if applicable)
![Screenshot 2023-04-20 at 13 29 36](https://user-images.githubusercontent.com/1729216/233366157-17a952c9-a781-4f90-ba77-25df5d4e149d.png)
